apps/api-example/src/main.ts

```diff
@@ -22,7 +24,22 @@ async function bootstrap() {
     credentials: true
   })
   nestApp.use(helmet())
-  nestApp.use(json({limit: MAX_PAYLOAD_SIZE}))
+
+  const skipPrefixes = [`/${MAIL_WEBHOOK_PATH_PREFIX}`, `/${PAYMENT_WEBHOOK_PATH_PREFIX}`] as const
+  const skipPrefixesLength = skipPrefixes.length
+  const jsonParser = json({limit: MAX_PAYLOAD_SIZE})
+
+  // Apply JSON parsing only when the path doesn't match any webhook prefix (plain for loop for maximum speed and minimal overhead)
+  const conditionalJson: RequestHandler = (req: Request, res: Response, next: NextFunction) => {
+    const path: string = req.path ?? req.url
+    for (let i = 0; i < skipPrefixesLength; i++) {
+      const p = skipPrefixes[i]
+      if (path === p || path.startsWith(p + '/')) return next()
```

Comment:
Maybe we can add `{}` around the return here so we don't repeat https://mfukar.github.io/2014/02/22/Apple-SSL-bug.html